### PR TITLE
Added: scripts for nightly/rc/release builds (SD card content, Russian voice), Audio normalization, PSV files for TTSAutomate

### PR DESCRIPTION
**Pull request by mr-pokryshkin**
Added: scripts for nightly build (SD card content, Russian voice), system sounds: done, other sounds: partially
Fixed: gTTS issue with unicode (Cyrillic) chars in command params (gTTS module instead of system command)

### CODE REVIEW

File: radio/util/tts_ru.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+                (u"Сенсор отключен", "sensorko", NO_ALTERNATE),
+                (u"Серва перегружена", "servoko", NO_ALTERNATE),
+                (u"Перегрузка", "rxko", NO_ALTERNATE),
+                (u"Приемник всё ещё подключен!", "modelpwr", NO_ALTERNATE),
+                ]:
+    systemSounds.append((s, filename(f, a)))
+
+for i, (s, f) in enumerate([
+                            (u"Аккумулятор садится!", "lowbat"),
+#                            (u"gear!, up!", "gearup"),
```
Comment (schwabe) [inline radio/util/tts_ru.py]: Lazyness or a good reason not include the optional sounds? :)

**Comment by schwabe** [comment]
Looks good, sound generation worked fine for me. Thanks!

**Comment by 3djc** [comment]
@bsongis please do not merge before 2.2.1, I would rather deal  with potential server impact in nightlies

**Comment by schwabe** [comment]
@3djc then we need to revert the other RU change for time being too, otherwise users will be able to select russian tts but we ship sdcard images without it. I am rather for merging this too.

**Comment by 3djc** [comment]
No, we have already 5 other languages that are supported, but for wich we do not supply language packs; so it would not be a first

**Comment by 3djc** [comment]
We could merge it since it applies only to nightly, not either release or rc, but the issue is that  to be run by the server, it would need sdcard version to be incremented, and that would be affecting rc/release too (and that would not generate sdpack for rc/release either). As soon as release is well released, we can merge than, I'll deal with server changes if any, and within one or two day, there will be a russian language pack available as a download. It should make our Russian comrades happy :)

File: radio/util/tts_ru.py (hunk 1 above)
Comment (mr-pokryshkin) [reply radio/util/tts_ru.py]: unfortunately, English texts sounds really weird, if I use Russian voice. So I decided to translate most important sounds, and later, after forum discussion, I'll update optional sounds.

**Comment by 3djc** [comment]
@mr-pokryshkin Could you complete the translations ? I still see a lot of english in there, then we can merge it

File: radio/util/tts_ru.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+for i, s in enumerate([u"одна", u"одиннадцать", u"двадцать одна", u"тридцать одна", u"сорок одна",
+                       u"пятьдесят одна", u"шестьдесят одна", u"семьдесят одна", u"восемьдесят одна", u"девяносто одна"]):
+    systemSounds.append((s, filename(PROMPT_SYSTEM_BASE + 180 + i)))
+
+# last digit = 2, female
+for i, s in enumerate([u"две", u"двенадцать", u"двадцать две", u"тридцать две", u"сорок две",
+                       u"пятьдесят две", u"шестьдесят две", u"семьдесят две", u"восемьдесят две", u"девяносто две"]):
+    systemSounds.append((s, filename(PROMPT_SYSTEM_BASE + 190 + i)))
+
+for i, (s, f) in enumerate([(u"вольт", "volt0"), (u"вольт", "volt1"),
```
Comment (bsongis) [inline radio/util/tts_ru.py:35]: don't we use python3? if yes those `u"..."` are not needed

**Comment by bsongis** [comment]
I don't know a single word of russian, so I think it's all good, just the python3 thing

**Comment by mpaperno** [comment]
I didn't go through every single string, but the translations (language-wise) looks good to me.   
Очень хорошо!  👍  А теперь Companion, да?  ;-)

**Comment by bsongis** [comment]
thanks!

**Comment by mr-pokryshkin** [comment]
I'm experienced .NET developer, but I'm newbie python developer. :) This is my first python script. So let me know about best practices or code style, I'll fix my files. If  u"" is optional, let me focus on new features and i'll fix it later.
My roadmap: 
1. Cyrilic fonts (for models naming)
2. Translate Open-Tx messages/menus etc
3. Companion

**Pull request merged**